sim/druid/swipe.go

```diff
@@ -5,31 +5,18 @@ import (
 
 	"github.com/wowsims/sod/sim/core"
 	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
 )
 
-const SwipeRanks = 5
-
-var SwipeSpellId = [SwipeRanks + 1]int32{0, 779, 780, 769, 9754, 9908}
-var SwipeBaseDamage = [SwipeRanks + 1]float64{0, 18, 25, 36, 60, 83}
-var SwipeLevel = [SwipeRanks + 1]int{0, 16, 24, 34, 44, 54}
-
 // See https://www.wowhead.com/classic/spell=436895/s03-tuning-and-overrides-passive-druid
 // Modifies Threat +101%:
-const SwipeThreatMultiplier = 2.0
+const SwipeThreatMultiplier = 3.5
 
 func (druid *Druid) registerSwipeBearSpell() {
 	hasImprovedSwipeRune := druid.HasRune(proto.DruidRune_RuneCloakImprovedSwipe)
+	baseMultiplier := 1.0
 
-	rank := map[int32]int{
-		25: 2,
-		40: 3,
-		50: 4,
-		60: 6,
-	}[druid.Level]
-
-	level := SwipeLevel[rank]
-	spellID := SwipeSpellId[rank]
-	baseDamage := SwipeBaseDamage[rank]
+	baseDamage := 83 + .1*druid.GetStat(stats.AttackPower)
```

Comment:
Why were the level-based ranks removed? Seems like there's no reason to have removed them if they were already in.